x-pack/plugins/reporting/server/lib/event_logger/logger.ts

```diff
@@ -24,7 +24,10 @@ import {
   StartedExecution,
 } from './types';
 
-/** @internal */
+export interface ExecutionClaimMetrics extends TaskRunMetrics {
+  queueDuration: number;
```

Comment:
nit; calling this `queueDurationMs` or `timeQueuedMs` might be a bit clearer. Otherwise a doc comment would do the trick!